AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/Connection/DataStoreConnectionScenario6Tests.swift

```diff
@@ -90,6 +90,119 @@ class DataStoreConnectionScenario6Tests: SyncEngineIntegrationTestBase {
         }
     }
 
+    func testGetCommentThenFetchPostThenFetchBlog() throws {
+        try startAmplifyAndWaitForSync()
+        guard let blog = saveBlog(name: "name"),
+              let post = savePost(title: "title", blog: blog),
+              let comment = saveComment(post: post, content: "content") else {
+            XCTFail("Could not create blog, post, and comment")
+            return
+        }
+
+        // fetch comment
+        let getCommentCompleted = expectation(description: "get comment complete")
+        var resultPost: Post6?
```

Comment:
this is confusing because the expectation is one thing and the result is something else, keeping it `resultComment` makes it clearer later especially since the test is making sure models are loaded, eagarly/lazily, etc. for example, is the `fetchedPost` not there because it is not eagarly loaded? or is it because the async call failed?